library-model/library-model-generator/src/main/java/com/uber/nullaway/libmodel/LibraryModelGenerator.java

```diff
@@ -189,7 +212,9 @@ private static class AnnotationCollectionVisitor extends VoidVisitorAdapter<Void
     private boolean isJspecifyNullableImportPresent = false;
     private boolean isNullMarked = false;
     private final Map<String, MethodAnnotationsRecord> methodRecords;
+    private final Set<String> nullMarkedClasses;
     private final Map<String, Set<Integer>> nullableUpperBounds;
+    // TODO this is sketchy!
```

Comment:
I added this comment 🙂 I think we need to do for arrays whatever JarInfer is now doing.  We should also add integration tests.